config/samples/clusterproviderdesc.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   name: default
 spec:
   ProviderType: "kind"
+  ClusterPrefixForDiscovery: "kubestellar"
```

Comment:
too long, and we want disconnect from kubestellar.  You can put "ks-"  Please add "-"